app/templates/merchant/view_listings.html

```diff
@@ -34,16 +33,19 @@
             <input value="{{ name_search_term }}"type="text" name="name-search" placeholder="e.g. John's corned beef">
         </div>
 
-        <div class="two fields advanced-search" style="{{ display }}">
-            <div class="three wide field">
-                <label>Minimum unit price</label>
-
-                <input value="{{ min_price }}" type="text" name="min-price" placeholder="e.g. $0">
-            </div>
-            <div class="three wide field">
-                <label>Maximum unit price</label>
-                <input value="{{ max_price }}" type="text" name="max-price" placeholder="e.g. $10">
+        <div class="twelve wide field advanced-search" style="{{ display }}">
+            <div class="two wide fields">
+                <div class="three wide field">
+                    <label>Minimum unit price</label>
+                    <input value="{{ min_price }}" type="number" id="min-price" name="min-price" placeholder="e.g. $0">
```

Comment:
To indicate that it's a price? Or do you want the slider to be specific enough to select within dollars.